usr/lib/api/policy.c

```diff
@@ -255,6 +255,7 @@ static CK_RV policy_extract_key_data(get_attr_val_f getattr, void *d,
         *comptarget = COMPARE_SYMMETRIC;
         break;
     case CKK_AES:
+    case CKK_AES_XTS:
         *siglen = 128;
         /* Fallthrough */
```

Comment:
As far as I understand the PKCS11 description of AES-XTS, `CKA_VALUE_LEN` is either 32 or 64, i.e., the length of the concatenation of the two keys.  To compute the strength, you have to consider the keys individually.  So you actually need to divide the computed size by 2 for AES-XTS.